### PR TITLE
Update 2021-07-01-havtorn21a.md

### PR DESCRIPTION
**Pull request by JakobHavtorn**
Corrected author names

### CODE REVIEW

File: _posts/2021-07-01-havtorn21a.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ lastpage: 4128
 page: 4117-4128
 order: 4117
 cycles: false
-bibtex_author: Havtorn, Jakob D. Drachmann and Frellsen, Jes and Hauberg, Soren and
+bibtex_author: Havtorn, Jakob D. and Frellsen, Jes and Hauberg, S{\o}ren and
   Maal{\o}e, Lars
 author:
-- given: Jakob D. Drachmann
-  family: Havtorn
+- given: Jakob
```
Comment (lawrennd) [inline _posts/2021-07-01-havtorn21a.md]:
```suggestion
- given: Jakob D.
```

File: _posts/2021-07-01-havtorn21a.md [hunk 2]
```diff
@@ -23,14 +23,14 @@ lastpage: 4128
...
+  family: Drachmann Havtorn
```
Comment (lawrennd) [inline _posts/2021-07-01-havtorn21a.md]:
```suggestion
  family: Havtorn
```

**Review by lawrennd — COMMENTED** [review]
Hi Jakob, Thanks for the fix. It was slightly inconsistent across the bibtex_author and author field. I've updated to match what the PDF says (where you are Jakob D. Havtorn) ... but that seems inconsistent with what you are proposing too! I.e.  is Drachmann part of your family name or is it one of your given names? Sorry for the mucking about! You can update my proposed changes as appropriate.

**Comment by JakobHavtorn** [comment]
Well, "Drachmann" is not part of my first name, "Havtorn" is definitely my last name. Some would call "Drachmann" a middle name. I regard it as part of my family name, being "Drachmann Havtorn", but that distinction is not too common in Denmark.

**Comment by JakobHavtorn** [comment]
The current edit should be good I suppose?

**Comment by lawrennd** [comment]
Merged ... let me know if that's OK!

**Pull request merged**